frameworks/Rust/axum/Cargo.toml

```diff
@@ -28,28 +28,63 @@ path = "src/main_mongo_raw.rs"
 name = "axum-pg"
 path = "src/main_pg.rs"
 
+[features]
+default = []
+simd-json = [
+    "dep:simd-json",
+    "dep:axum-core",
+    "dep:mime",
+    "dep:bytes",
+    "dep:serde_path_to_error",
+]
+
 [dependencies]
-axum = { version = "0.6.16", default-features = false, features = ["json", "query", "http1", "tokio"] }
-deadpool = { version = "0.10.0", features = ["rt_tokio_1", "serde", "async-trait", "managed" ] }
-deadpool-postgres = "0.12.1"
+axum = { version = "0.7.5", default-features = false, features = [
+    "json",
+    "query",
+    "http1",
+    "tokio",
+] }
+deadpool = { version = "0.12.1", features = ["rt_tokio_1", "serde", "managed"] }
+deadpool-postgres = { version = "0.14.0", features = ["rt_tokio_1", "serde"] }
 dotenv = "0.15.0"
-futures = "0.3.25"
-futures-util = "0.3.25"
-hyper = { version = "0.14.23", features = ["http1", "server"] }
-mongodb = { version = "2.3.1", features = ["zstd-compression", "snappy-compression", "zlib-compression"] }
-num_cpus = "1.14.0"
+futures = "0.3.30"
+futures-util = "0.3.30"
+mongodb = { version = "2.8.0", features = [
+    "zstd-compression",
+    "snappy-compression",
+    "zlib-compression",
+] }
+num_cpus = "1.16.0"
 rand = { version = "0.8.5", features = ["small_rng"] }
-serde = { version = "1.0.149", features = ["derive"] }
-serde_json = "1.0.89"
-sqlx = { version = "0.7.3", features = ["postgres", "macros", "runtime-tokio-native-tls"] }
-tokio = { version = "1.24.2", features = ["full"] }
-tokio-pg-mapper = "0.2.0"
-tokio-pg-mapper-derive = "0.2.0"
-tokio-postgres = "0.7.7"
-tower = { version = "0.4.13", features = ["util"] }
-tower-http = { version = "0.4.0", features = ["set-header"] }
+serde = { version = "1.0.196", features = ["derive"] }
+serde_json = "1.0.127"
+sqlx = { version = "0.7.3", features = [
+    "postgres",
+    "macros",
+    "runtime-tokio",
+    "tls-rustls",
+] }
+tokio = { version = "1.39.3", features = ["full"] }
+tokio-pg-mapper = { version = "0.2.0" }
+tokio-pg-mapper-derive = { version = "0.2.0" }
+tokio-postgres = { version = "0.7.11" }
+tower = { version = "0.5.0", features = ["util"] }
+tower-http = { version = "0.5.2", features = ["set-header"] }
 yarte = "0.15.7"
+simd-json = { version = "0.13.8", optional = true }
+axum-core = { version = "0.4.3", optional = true }
+mime = { version = "0.3.17", optional = true }
+bytes = { version = "1.5.0", optional = true }
+serde_path_to_error = { version = "0.1.15", optional = true }
+moka = { version = "0.12.8", features = ["future"] }
+socket2 = "0.5.7"
+hyper = { version = "1.4", features = ["server", "http1"] }
+hyper-util = { version = "0.1", features = ["tokio", "server-auto", "http1"] }
+
 
 [profile.release]
-lto = true
+lto = "fat"
 codegen-units = 1
+strip = true
+opt-level = 3
```

Comment:
`opt-level = 3` is already default for the release profile: [docs](https://doc.rust-lang.org/cargo/reference/profiles.html#release)